www/includes/easyparliament/templates/html/index.php

```diff
@@ -93,6 +94,7 @@
             </div>
         </div>
 
+        <?php if ( $featured_debate_shown == false ) { ?>
```

Comment:
Should this if go round just the lines 101-105, otherwise the topics aren't shown if featured has been shown above?

Reply:
Yep, can do that.